docs/arc42/context_view.md

```diff
@@ -0,0 +1,142 @@
+# Context View
+
+## System Context
+
+The geOrchestra Gateway is a central component in the geOrchestra Spatial Data Infrastructure (SDI), acting as the primary entry point for all user interactions with the platform.
+
+### System Context Diagram
+
+The system context diagram below shows the Gateway in the context of the overall geOrchestra system and its external dependencies.
+
+![System Context Diagram](../assets/images/structurizr/structurizr-SystemContext.svg)
+
+The diagram shows:
+
+1. **Users** access the geOrchestra platform through the Gateway using HTTPS
+2. **External Identity Providers** can authenticate users via OAuth2/OpenID Connect
+3. **geOrchestra Gateway** acts as the entry point for all services
```

Comment:
not really shown actually

Reply:
I've updated the context_view.md document to correctly reflect what's actually shown in the system context diagram.
* [Fix](https://github.com/georchestra/georchestra-gateway/compare/b0de5620faab254021e15451c9beb441fb941d77..238db3b807f84932d4130ff5ff556dde2be8bfaf)
* [Live](https://groldan.github.io/georchestra-gateway/arc42/context_view/#system-context-diagram)

Reply:
wouldn't it rather be the gateway intead of thegeOrchestra SDI ?